srcStatic/OP2Memory.cpp

```diff
@@ -6,7 +6,7 @@
 #include <iomanip>
 
 
-bool memoryCommandsDisabled;
+bool memoryPatchingEnabled = false;
```

Comment:
I like changing this variable from Disabled to Enabled. Makes the code easier to read.